### PR TITLE
PayPal Web Integration

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - This pr adds the paypal web fallback in a separate module

### Authors

- @jaxdesmarais
- @sshropshire

### CODE REVIEW

File: Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import UIKit
-import PayPal
+import PayPalWebCheckout
 import PaymentsCore
 
 class PayPalDemoViewController: FeatureBaseViewController {
```
Comment (jaxdesmarais) [inline Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift]: Should we rename this `PayPalWebCheckoutViewController` and have a seperate `PayPalNativeCheckoutViewController` to demo the PayPal checkout SDK?

Comment (jaxdesmarais) [reply Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift]: I guess if the NativeCheckout isn't being released, probably don't need to add it to the demo, but we should rename this ViewController.

**Review by jaxdesmarais — APPROVED** [review]
This looks good. That's for seperating out the modules/renaming/pulling in the code from #36!

One small comment but otherwise 🚢

File: Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift (hunk 1 above)
Comment (jaxdesmarais) [reply Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift]: We should also change the demo type from PayPal > PayPal Web Checkout

File: Demo/Demo/DemoSettings/DemoType.swift [hunk 2]
```diff
@@ -3,24 +3,24 @@ import SwiftUI
 
 enum DemoType: String {
     case card
-    case paypal
+    case payPalWebCheckout
```
Comment (sshropshire) [inline Demo/Demo/DemoSettings/DemoType.swift:6]: There's some inconsistency with this naming convention. I'm wondering is `paypal` or `payPal` a better prefix when at the beginning of a camel case variable?

Comment (jaxdesmarais) [reply Demo/Demo/DemoSettings/DemoType.swift:6]: I think we want to do `payPal` - I feel like we talked about this with Sammy :pour-one-out: at some point and all agreed on that.

Comment (sshropshire) [reply Demo/Demo/DemoSettings/DemoType.swift:6]: Nice aight. I just fixed camel casing throughout to keep it consistent.

**Review by jcnoriega — APPROVED** [review]

**Pull request merged**